tools/setup_helpers/generate_code.py

```diff
@@ -197,7 +197,7 @@ def main() -> None:
         assert os.path.isfile(ts_backend_yaml), f"Unable to access ts_backend_yaml: {ts_backend_yaml}"
         assert os.path.isfile(ts_native_functions), f"Unable to access {ts_native_functions}"
         from tools.codegen.gen_lazy_tensor import run_gen_lazy_tensor
-        from tools.codegen.dest.lazy_ir import TSLazyIR
+        from tools.codegen.dest.lazy_ir import GenTSLazyIR
         run_gen_lazy_tensor(aten_path=aten_path,
```

Comment:
Just looking around - is `gen_ts_lowerings` still used anywhere? it looks like in `gen_lazy_tensor.py`, it's  used to implicitly reference `GenTSLazyIR`, but now we explicitly pass that class in here instead. 

Do you also need a way to tell the codegen to override the `GenLazyNativeFuncDefinition` abstract class? (Maybe we should also try to bundle them together so there's one place where the extensions live).